src/appleseed/renderer/kernel/intersection/embreescene.h

```diff
@@ -96,6 +97,9 @@ class EmbreeScene
   private:
     struct Impl;
     Impl* impl;
+
```

Comment:
Is the private implementation still useful/reasonable at this point?

Reply:
Removed